cli/src/main/java/bisq/cli/CliMain.java

```diff
@@ -88,7 +92,10 @@
         createpaymentacct,
         getpaymentaccts,
         getversion,
-        getbalance,
+        @Deprecated getbalance, // Use getbalances, return bsq and btc balance info
```

Comment:
Let's remove it

Reply:
> Let's remove it

This has already been replaced in one of the next PRs. Commit 9d863e5 removed the deprecated core api method.